core/src/avm2/multiname.rs

```diff
@@ -220,12 +221,37 @@ impl<'gc> Multiname<'gc> {
         })
     }
 
+    #[inline(never)]
+    #[cold]
+    pub fn try_replace_with_qname(
+        &self,
+        obj: Object<'gc>,
+        activation: &mut Activation<'_, 'gc, '_>,
+    ) -> Option<Self> {
+        if let Object::QNameObject(qname_object) = obj {
+            if self.has_lazy_ns() {
+                activation.pop_stack(); // ignore the ns component on stack
```

Comment:
style nit, `let _ = activation.pop_stack();` is a little more explicit about ignoring this

Reply:
In that case, should we add `#[must_use]` to the function?

Reply:
Yep! I agree